pdal/util/Bounds.hpp

```diff
@@ -307,7 +307,7 @@ class PDAL_DLL BOX2D
 /**
   BOX3D represents a three-dimensional box with double-precision bounds.
 */
-class PDAL_DLL BOX3D : private BOX2D
+class PDAL_DLL BOX3D : public BOX2D
```

Comment:
I tend to thing that all these BOX4D changes should ride with TileDB since I don't see them being used generically. @hobu?

Reply:
IDK. Are we ever going to add time indexing to PDAL? I kind of thing not.

Reply:
I think that when there is more need, it can move. I don't see any reason for this in a public interface.